pdns/communicator.hh

```diff
@@ -46,22 +46,36 @@ struct SuckRequest
   DNSName domain;
   ComboAddress master;
   bool force;
+  uint8_t priority;
```

Comment:
YOu might want to make these a pair, so you don't have to define a custom compare function

Reply:
Changed it to a pair :D